bears/natural_language/LanguageToolBear.py

```diff
@@ -57,12 +59,19 @@ def run(self,
         natural_language = (guess_language(joined_text)
                             if natural_language == 'auto'
                             else natural_language)
-        natural_language = 'en-US' if not natural_language \
-                           else natural_language
 
-        tool = LanguageTool(natural_language, motherTongue='en_US')
-        tool.disabled.update(languagetool_disable_rules)
+        try:
+            tool = LanguageTool(natural_language, motherTongue='en_US')
+        # Using 'en-US' if guessed language is not supported
```

Comment:
I would put the comment into the `except`-block, so just move it one line down :) That's a more common placement :)